src/model/model.cpp

```diff
@@ -0,0 +1,83 @@
+#include <stack>
+#include <unordered_set>
+
+#include <deepworks/model.hpp>
+
+#include "expression/call_impl.hpp"
+#include "expression/placeholder_impl.hpp"
+
+#include "model/model_impl.hpp"
+#include "model/graphbuilder.hpp"
+
+#include "util/assert.hpp"
+
+using namespace deepworks::graph;
+
+deepworks::Model::Model(deepworks::Placeholder in, deepworks::Placeholder out)
+    : deepworks::Model(deepworks::Placeholders{in}, deepworks::Placeholders{out}) {
```

Comment:
Should we std::move arguments ?

Reply:
Probably yes, but anyway copy is free. Ok will add move

Reply:
Done